.github/workflows/bindings-nodejs-publish.yml

```diff
@@ -0,0 +1,181 @@
+name: Node.js publish to NPM
+
+on: workflow_dispatch
+
+env:
+  CARGO_INCREMENTAL: 0
+
+jobs:
+  publish-nodejs:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: bindings/nodejs
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Rust
+        uses: ./.github/actions/setup-rust
+
+      # Required for ledger-nano
+      - name: Install required packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install libudev-dev libusb-1.0-0-dev
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "18.x"
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Install Yarn
+        run: npm i -g yarn
+
+      - name: Install JS dependencies
+        run: yarn
+
+      - name: Build project
+        run: yarn build
+
+      - name: Update Cargo.toml with git
+        run:  sed -i 's#path = "../core"#git = "https://github.com/iotaledger/iota-sdk", rev = "'$GITHUB_SHA'"#g' Cargo.toml 
```

Comment:
I get this locally 
```
bash-3.2$ sed -i 's#path = "../core"#git = "https://github.com/iotaledger/iota-sdk", rev = "'$GITHUB_SHA'"#g' Cargo.toml 
sed: 1: "Cargo.toml": invalid command code C
```

Reply:
Maybe something macos specific? I tried it here https://github.com/Thoralf-M/iota-sdk/actions/runs/5311626540/jobs/9615082906 and there it worked

Reply:
Ok fair enough